components/commentsListView.html

```diff
@@ -1,3 +1,52 @@
-<li ng-repeat="comment in ctrl.comments" ng-show="!comment.isRemoved" class="ng-hide">
-	<input type="checkbox">{{comment.text}}
-</li>
\ No newline at end of file
+<!--Почему-то не работало свойство checked-->
+<label>
```

Comment:
Could you please use ng-repeat directive for implementation of radio-button. Use ng-value for set value, ng-change for update comments type, and ng-model is selectedType that you'll add in your controller